src/FakeItEasy/Fake.cs

```diff
@@ -71,15 +71,31 @@ public static void ClearRecordedCalls(object fakedObject)
         /// <summary>
         /// Gets the fake manager associated with the proxy, if any.
         /// </summary>
-        /// <param name="fakedObject">The proxy to get the manager from.</param>
-        /// <returns>The fake manager, or <c>null</c> if <paramref name="fakedObject"/> is not actually a faked object.</returns>
+        /// <param name="potentialFake">The potential proxy to get the manager from.</param>
+        /// <param name="fakeManager">The fake manager, or <c>null</c> if <paramref name="potentialFake"/> is not actually a faked object.</param>
+        /// <returns><c>true</c> if <paramref name="potentialFake"/> is a faked object, else false.</returns>
         [DebuggerStepThrough]
         [SuppressMessage("Microsoft.Naming", "CA1720:IdentifiersShouldNotContainTypeNames", MessageId = "object", Justification = "The term fake object does not refer to the type System.Object.")]
-        internal static FakeManager TryGetFakeManager(object fakedObject)
+        public static bool TryGetFakeManager(object potentialFake, out FakeManager fakeManager)
         {
-            Guard.AgainstNull(fakedObject, nameof(fakedObject));
+            Guard.AgainstNull(potentialFake, nameof(potentialFake));
+
+            fakeManager = FakeManagerAccessor.TryGetFakeManager(potentialFake);
+            return fakeManager != null;

```

Comment:
In #1624 we adopted a system where a null check is either `blah is null` or `blah is object`. We should note it up in the docs about how to contribute. Could you update to `return fakeManager is object`?